kit/ChildSession.cpp

```diff
@@ -543,9 +543,17 @@ bool ChildSession::_handleInput(const char *buffer, int length)
             }
             else if (tokens[1].find(".uno:Save") != std::string::npos)
             {
-                // Disable processing of other messages while saving document
-                InputProcessingManager processInput(getProtocol(), false);
-                return unoCommand(tokens);
+                static bool doBgSave = !!getenv("COOL_BGSAVE");
```

Comment:
Would we have a separate command for sync-saving when cleaning up DocBroker?

Reply:
Ah perhaps - we want a parameter here too for the save uno command to enable this code-path I think; TBD.